File: dictionaries/dict_comprehension/preise_halbieren.py
# Preise halbieren
preise = {
    "Apfel": 1.20,
    "Banane": 0.80,
    "Kirsche": 2.50
}
preise_halbiert = {obst: preis / 2 for obst, preis in preise.items()}
print(f"Halbierte Preise: {preise_halbiert}")

File: Vergleich_mit_JS/preiseHalbieren.js
// Preise halbieren
const preise = {
  Apfel: 1.2,
  Banane: 0.8,
  Kirsche: 2.5,
};

const preiseHalbiert = Object.fromEntries(
  Object.entries(preise).map(([obst, preis]) => [obst, preis / 2])
);
console.log(preiseHalbiert);
